packages/rlc-common/src/interfaces.ts

```diff
@@ -123,6 +123,7 @@ export interface RLCOptions {
   azureArm?: boolean;
   sourceFrom?: "Cadl" | "Swagger";
   isModularLibrary?: boolean;
+  sourceOnly?: boolean;
```

Comment:
I assume we could disable the option `generateMetadata` and then we could only generate src folder.

Reply:
`generateMetaData: false` should equals `sourceOnly: true`

Reply:
Oh yeah, got your point, because we are putting the generated code into sources folder. Thanks for the explain :)